### PR TITLE
youtube channelas name under youtube Channels as pe #1

### LINKED ISSUE #1: Add resource Good Youtube channels list under Youtube Channels Category
Request to Add category and Add resource,

Name : FreeCodeCamp
URL : https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ
Description :  In my opinion this is one the best Channel on youtube
Category : Youtube Channel
Rate plateform/tools :  10 out of 10

- [ ✅ ] I've read the CONTRIBUTING.md
- [ ✅ ] I've used the mentioned resource.

### PR DESCRIPTION
**Pull request by MananDesai54**
#### Changes done,
-  In youTube channel catagory
-  Added 2 youtube channels

Fix #1 

Kindly Review.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
```
Comment (Hardik0307) [inline .github/pull_request_template.md]: Redo changes in ```pull_request_template.md```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 
 #### Changes done,
--
-- 
+- In Youtube catagory
```
Comment (rohitkg98) [inline .github/pull_request_template.md]: I do no think this file is meant to be changed

**Review by Hardik0307 — COMMENTED** [review]
Just check that.

File: .github/pull_request_template.md (hunk 2 above)
Comment (Hardik0307) [reply .github/pull_request_template.md]: exactly @rohitkg98

File: .github/pull_request_template.md (hunk 1 above)
Comment (MananDesai54) [reply .github/pull_request_template.md]: okay

Comment (MananDesai54) [reply .github/pull_request_template.md]: Check now

**Review by Hardik0307 — APPROVED** [review]

**Comment by Hardik0307** [comment]
LGTM, Great Work @MananDesai54.

**Pull request merged**